b9/src/VirtualMachine.cpp

```diff
@@ -99,12 +99,13 @@ std::size_t VirtualMachine::getFunctionCount() {
 
 void VirtualMachine::generateAllCode() {
   assert(cfg_.jit);
-  auto functionIndex = 0;
+  compiledFunctions_.push_back(nullptr); //TODO: <body> function right now is "compiled" as null
+  auto functionIndex = 1; //0 index for <body>
```

Comment:
I would rather fix the compilation issue in this PR, than skip over compiling <body>.

Reply:
I agree.